llvm/lib/Transforms/Vectorize/VPlanTransforms.h

```diff
@@ -174,7 +175,8 @@ struct VPlanTransforms {
   static void handleUncountableEarlyExit(VPlan &Plan, ScalarEvolution &SE,
                                          Loop *OrigLoop,
                                          BasicBlock *UncountableExitingBlock,
-                                         VPRecipeBuilder &RecipeBuilder);
+                                         VPRecipeBuilder &RecipeBuilder,
+                                         VFRange &Range);
```

Comment:
nit: Range is not modified by this function, so better to make it const.
```suggestion
                                         const VFRange &Range);
```

Reply:
Done, thanks!